temfpy/uncertainty_quantification.py

```diff
@@ -56,18 +56,28 @@ def eoq_model(x, r=0.1):
     and the setup costs of an order `x[2]`. The annual interest rate `r` is treated as an
     additional parameter.
 
+    .. math::
+        y = \sqrt{\frac{24 x_0 x_2}{r x_1}}
+
+    .. figure:: ../../docs/_static/images/fig-eoq-tradeoff.png
```

Comment:
Please also add a folder where we have the source code that creates the figure ... that also includes a proper matplotlibrc file.